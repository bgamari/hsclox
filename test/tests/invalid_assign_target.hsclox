1*4="hi";
