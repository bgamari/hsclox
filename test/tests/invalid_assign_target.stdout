[line 1] Error at '=': Invalid assignment target.
